### ISSUE TITLE
Add test for mini batch (gradient accumulation) for forge-fe vs torch

### ISSUE THREAD

**Issue by ndrakulicTT** [tenstorrent/tt-blacksmith#50]
Similar test to `tt-forge-fe/forge/test/mlir/mnist/training/test_training.py::test_mnist_training` just with `batch_size>1`

**Comment by skhatriTT** [comment]
@nvukobratTT do you know if it's still active or has been worked on?

**Comment by ndrakulicTT** [comment]
Closing as stale issue, and currently out of scope as ffe is deprecated
